abaco.go

```diff
@@ -409,6 +411,8 @@ func NewAbacoSource() (*AbacoSource, error) {
 	source.arings = make(map[int]*AbacoRing)
 	source.groups = make(map[GroupIndex]*AbacoGroup)
 	source.channelsPerPixel = 1
+	source.unwrapEnable = true
+	source.unwrapResetSamp = 20000
```

Comment:
You already have these same default values in `rpc_server.go`, so I think the ones here are never used since they should immediately  be overwritten? I suggest either removing them or setting them to more obviously wrong values like `-100000`. In either case add a comment that the values are expected to be overwritten by the configure function.

Reply:
Will remove.